bring/src/main/java/com/bobocode/petros/bring/context/AnnotationConfigApplicationContext.java

```diff
@@ -68,11 +74,19 @@ public <T> T getBean(String name, Class<T> requiredType) {
 
     void registerBeanDefinitions(final Set<Class<?>> allClasses) {
         Objects.requireNonNull(this.configurationBeanDefinitionScanner, "Configuration scanner must be initialized!");
-        var configurationBeanDefinitions = this.configurationBeanDefinitionScanner.scan(allClasses);
-
+        Objects.requireNonNull(this.componentsBeanDefinitionScanner, "Component scanner must be initialized!");
+        var beanDefinitions = getBeanDefinitions(allClasses);
         var registry = DefaultBeanDefinitionRegistry.getInstance();
-        configurationBeanDefinitions.forEach(
+
+        beanDefinitions.forEach(
                 beanDefinition -> registry.registerBeanDefinition(beanDefinition.getBeanName(), beanDefinition)
         );
     }
+
+    private List<BeanDefinition> getBeanDefinitions(Set<Class<?>> classes) {
```

Comment:
This method and registering logic can be simplified as following:
`Stream.concat(this.configurationBeanDefinitionScanner.scan(allClasses).stream(),
                        this.componentsBeanDefinitionScanner.scan(allClasses).stream())
                .forEach(beanDefinition -> registry.registerBeanDefinition(beanDefinition.getBeanName(), beanDefinition));`